src/utils/plotSpecs/generateEmbeddingCategoricalSpec.js

```diff
@@ -196,7 +196,7 @@ const generateSpec = (config) => {
   };
 };
 
-const generateData = (spec, cellSets, selectedCellSet, embeddingData) => {
+const filterCells = (cellSets, selectedCellSet) => {
```

Comment:
Created new function to facilitate checking for only one sample in `EmbeddingPreview`